signac/contrib/schema.py

```diff
@@ -70,17 +70,17 @@ def _build_job_statepoint_index(exclude_const, index):
             if key == "_id" or key.split(".")[0] != "statepoint":
                 continue
             collection.index(key, build=True)
-    tmp = collection._indexes
+    indexes = collection._indexes
 
     def strip_prefix(key):
-        return k[len("statepoint.") :]
+        return key[len("statepoint.") :]
```

Comment:
This change fixes a bug that only happened to work because the variable below was named `k`. It was being pulled from the enclosing scope (though it is unbound at the time of the function definition), rather than using the provided argument `key`.